### PR TITLE
Remove trip-data from Pulsar Properties and put it to payload

### PR DESCRIPTION
**Pull request by paasovaara**
Breaking change!! 

related to this: https://github.com/HSLdevcom/transitdata-tripupdate-processor/issues/40
requires merging and releasing new commons version: https://github.com/HSLdevcom/transitdata-common/pull/63

### CODE REVIEW

**Review by kenkus-futurice — APPROVED** [review]
LGTM.

File: src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java [hunk 1]
```diff
@@ -133,77 +142,48 @@ public void setLastModifiedTimeStamp(long ts) {
...
-            routeName = map.get(TransitdataProperties.KEY_ROUTE_NAME);
-            startTime = map.get(TransitdataProperties.KEY_START_TIME);
-            operatingDay = map.get(TransitdataProperties.KEY_OPERATING_DAY);
-            if (!isValid()) {
-                //Let's print more info for debugging purposes:
-                log.warn("JourneyInfo is missing some fields. Content: " + this.toString());
+    protected Optional<PubtransTableProtos.DOITripInfo> getTripInfo(long dvjId, long jppId) {
+        try {
+            String stopIdKey = TransitdataProperties.REDIS_PREFIX_JPP + Long.toString(jppId);
+            Optional<String> maybeStopId = Optional.ofNullable(jedis.get(stopIdKey));
```
Comment (kenkus-futurice) [inline src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java]: Could these redis calls be wrapped?

Comment (kenkus-futurice) [reply src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java]: What are JPP and DVJ btw?

File: src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java [hunk 2]
```diff
@@ -133,77 +142,48 @@ public void setLastModifiedTimeStamp(long ts) {
...
-
-        @Override
-        public String toString() {
-            return new StringBuilder()
-                    .append("direction: ").append(direction)
-                    .append(" routeName: ").append(routeName)
-                    .append(" startTime: ").append(startTime)
-                    .append(" operatingDay: ").append(operatingDay)
-                    .toString();
+        catch (Exception e) {
```
Comment (kenkus-futurice) [inline src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java:181]: So this is thrown if some tripinfo fields are missing and `builder.build()` fails?

Comment (paasovaara) [reply src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java:181]: yes, builder.build throws a runtime exception if the data is not valid or required fields are not set. I don't really like this so I have been explicitly adding throw-clauses to some factory-methods that do this.

File: src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java (hunk 1 above)
Comment (paasovaara) [reply src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java]: JourneyPatternPoint, is it perhaps a unique stop identifier? dunno, @laaksma ? dvj-id is the trip id.

Comment (paasovaara) [reply src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java]: not sure if I understand the redis wrapping...? to some class / interface?

Comment (kenkus-futurice) [reply src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java]: Yes, utility functions. Although not necessary if redis is not used elsewhere?

Comment (paasovaara) [reply src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java]: good idea, done.

**Pull request merged**